app/src/main/java/com/pysiak/simple/clock/ui/TabFragment.java

```diff
@@ -0,0 +1,19 @@
+package com.pysiak.simple.clock.ui;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import com.pysiak.simple.clock.R;
+
+public class TabFragment extends Fragment {
+    @Nullable
```

Comment:
@Nullable - remove thinks form the Internet you don't know how to use and why....